m4_sessions/1711-inning/Group_1.md

```diff
@@ -12,11 +12,45 @@ How to act and what to do fix it?
 
 Session Outline: Include the flow of the session and the time for each section.
 
-0 - 15 minutes: Everyone introduction
-15-25: video & article
-25-40: small group discussion
-40-50: large group discussion
-50-75: Talk about the future and actions we can do 
+#### 0 - 15 minutes: Everyone introduction
+* We each introduce ourselves
+* We have everyone go around and say what their favorite thing to do on the internet is
+#### 15-25: video & article
+* 7 minutes of this video by John Oliver https://www.youtube.com/watch?v=92vuuZt7wak
+* https://www.inverse.com/article/39338-net-neutrality-startups-innovation-fcc 
+#### 25-35: large group discussion
+* The history of telecom regulation - Ben lead
+* Does everyone understand net neutrality? - Team facilitate this discussion as a whole
+* Anyone have questions?
+* Everyone install net neutrality chrome extension and see how it affects users? - Johanan Lead  https://chrome.google.com/webstore/detail/removal-of-net-neutrality/macmdnlopncdoehmjhfenfblflnohoen?hl=en-US
+#### 35-50: Small group discussion to answer questions - Put questions on board. Number off to 4.
+* Show 3 bullet points of Net Neutrality on the board 
+  - BLOCKING Internet service providers could not discriminate against any lawful content by blocking websites or apps.
+  - THROTTLING Service providers could not slow the transmission of data based on the nature of the content, as long as it is legal.
+  - PAID PRIORITIZATION Service providers could not create an internet fast lane for companies and consumers who pay premiums, and a slow lane for those who don’t.
+* How will this affect big businesses versus small businesses? 
+  - Fast lanes
+  - Big businesses paying for priority
+  - Entreprenuership
+* How will this affect us as consumers? - Entire Team Study
+  - ISP throttling
+  - Want to access Facebook and Twitter? Under a bundling system, getting on those sites could require paying for a premium social media package.
+  - Could control politics that benefit them, or companies could pay them to promote or even censor political ideas.  If 
+  comcast or any other ISPs are pushing an agenda or taking money from a campaign and the ISP slows down the opposing party.
+* How do you see this affect us as developers? - Entire Team Study
+   - Hurt developers looking to create start ups
+   - Need more talking points here
+* Who else will this affect?
+* Show this image on the 
+* Who does this benefit? Yohanan hit on why certain people want this.
+#### 50-60: Discuss group answers.  Show this image on the tv https://pbs.twimg.com/media/DNGlrABUIAAr9RO.jpg:large
+#### 60-75: Talk about the future and actions we can do 
+* When will we start seeing the affects of this?
+* Congressional Review Act - Robbie lead
+* ACLU, States and others filing lawsuits against FCC - Ben lead
+* Write your representatives 
+* Voting in Upcoming Elections - Danny look into more actions
```

Comment:
This is great!  Really great actions.  Maybe put these in the slack channel so that students can take action immediately.